src/main/steps/common/page.njk

```diff
@@ -64,9 +67,17 @@
       {% endblock %}
     </div>
     <div class="govuk-grid-column-one-third">
-        {% block page_contact_us %}
```

Comment:
@hmcts-abduldaim Again these are common changes, not sure why you removed dynamic page link & added hardcoded contact for help details.
Please revert, let's try reuse the same.